README.md

```diff
@@ -3,10 +3,32 @@
 This project is generated with [yo angular generator](https://github.com/yeoman/generator-angular)
 version 0.15.1.
 
+## Getting Started
+1. git clone
+2. cd into the folder `cd arthop-map-dev`
+3. check node version `node --version`
+  * If node < 6.4.0 install using nvm
+  * `nvm use 6.4.0` or `nvm install 6.4.0` 
+  * if nvm is not installed on your machine
+    * Installation instructions [bash] (https://github.com/creationix/nvm) [zsh] (https://github.com/lukechilds/zsh-nvm#manually)
+4. Make sure nodemon is installed `nodemon --version`
+  * To install run `npm install -g nodemon`
+5. Run `npm install` to install dependencies
+6. Run `cd app/public/ && bower install`
+  * Remember to `cd ../..` to get back to root directory
+  * If bower is not globally installed run `npm install -g bower`
```

Comment:
My suggestion would be to Add a Dependencies section to the docs. Something like:

## Dependencies
* Node 6.4 or higher
* MongoDB install
* Global install of Bower and nodemon `npm install -g nodemon bower`